ci_framework/roles/polarion/molecule/default/converge.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  tasks:
+    - name: Do not do anything
+      debug:
+        msg: |
+          This role mostly has no ansible tasks, most if relying on go.
```

Comment:
this isn't the actual reason :).

More like "this role relies on a polarion instance, and CI doesn't provide anything".

Note that, maybe, we can get something ensuring it's bootstraping the role correctly, such as:
- ensure venv is properly created
- ensure it can scrap files accordingly
- ensure it may be able to install dependencies
- and so on

This would require some `dry_run` parameter that may be used in order to deactivate some of the tasks.

Reply:
I don't think that would be possible to achieve, the tasks are dependent on internal repository and a molecule could not run on public to test it.
Unless I'm wrong on where/how molecule tests being run.

Reply:
ah, if even packages are only internals, indeed. Still - please update the message in order to point to the actual reason (the ones you listed here ;)).

Reply:
ack, thanks.